### PR TITLE
Add documentation for namespace annotations

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

This pull request adds documentation for namespace annotations, feature added in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3684

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/kubernetes/best-practices.md [hunk 1]
```diff
@@ -348,6 +364,9 @@ The `sumologic.com/sourceCategoryReplaceDash` annotation with value `-` prevents
 ### Excluding data using annotations
 
 You can use the `sumologic.com/exclude` [annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) to exclude data from Sumo. This data is sent to the metadata enrichment service, but not to Sumo.
+You can exclude data per pod using pod annotations or per namespace using namespace annotations but pod annotations take precedence over namespace annotations.
+
+For example using pod annotations:
```
Comment (sumo-drosiek) [inline docs/send-data/kubernetes/best-practices.md:369]: so if I exclude namespace, I can still get data for pod using annotations?

Comment (kkujawa-sumo) [reply docs/send-data/kubernetes/best-practices.md:369]: yes, tested with this configuration

```
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    sumologic.com/exclude: "true"
  name: test-exclude
---
apiVersion: v1
kind: Pod
metadata:
  name: pod-a
  namespace: test-exclude
spec:
  containers:
    - name: container1
      image: public.ecr.aws/docker/library/busybox:latest
      args:
        - /bin/sh
        - -c
        - |
          while true; do
            echo "2022-02-08T20:53:28.621+0000 1st single line... pod-a"
            echo "2022-02-08T20:53:29.622+0000 2nd single line... pod-a"
            echo "2022-02-08T20:53:29.622+0000 3nd single line... pod-a"
            sleep 5
          done
---
apiVersion: v1
kind: Pod
metadata:
  name: pod-b
  namespace: test-exclude
  annotations:
    sumologic.com/include: "true"
spec:
  containers:
    - name: container1
      image: public.ecr.aws/docker/library/busybox:latest
      args:
        - /bin/sh
        - -c
        - |
          while true; do
            echo "2022-02-08T20:53:28.621+0000 1st single line... pod-b"
            echo "2022-02-08T20:53:29.622+0000 2nd single line... pod-b"
            echo "2022-02-08T20:53:29.622+0000 3nd single line... pod-b"
            sleep 5
          done

```

Comment (sumo-drosiek) [reply docs/send-data/kubernetes/best-practices.md:369]: I would mention this example explicitly 😅

**Review by kimsauce — APPROVED** [review]

File: docs/send-data/kubernetes/best-practices.md [hunk 2]
```diff
@@ -348,6 +365,9 @@ The `sumologic.com/sourceCategoryReplaceDash` annotation with value `-` prevents
 ### Excluding data using annotations
 
 You can use the `sumologic.com/exclude` [annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) to exclude data from Sumo. This data is sent to the metadata enrichment service, but not to Sumo.
+You can exclude data per pod using pod annotations or per namespace using namespace annotations but pod annotations take precedence over namespace annotations.
```
Comment (kimsauce) [inline docs/send-data/kubernetes/best-practices.md:368]:
```suggestion
You can exclude data per pod using pod annotations or per namespace using namespace annotations. However, pod annotations take precedence over namespace annotations.
```

**Pull request merged**